src/CET.cpp

```diff
@@ -56,7 +56,7 @@ CET::CET()
     : m_options(m_paths)
     , m_bindings(m_paths)
     , m_window(&m_overlay, &m_bindings, &m_d3d12)
-    , m_d3d12(m_window, m_paths, m_options)
+    , m_d3d12(m_window, m_paths, m_options, m_bindings)
```

Comment:
Instead of doing this, you could just register a function in m_d3d12.OnUpdate ?

Reply:
hope that is better now :P 
didnt know where to put it before... but i think this would be the most proper place for it.
dont know if you like my approach though. found it better than having this connect performed in CET master